scss/tech.scss

```diff
@@ -0,0 +1,63 @@
+%tech-card-item {
+  color: $mine-shaft;
+  font-family: $font-stack;
+  font-size: $text-medium;
+  text-align: center;
+}
+
+.tech-list .tech-card {
+  margin: 30px 20px;
+}
+
+.tech-card::before {
+  border: 1px solid $earls-green;
+  border-radius: $tech-card-border-radius;
+  content: '';
+  height: 100%;
+  left: -$tech-card-box-overlap;
+  position: absolute;
+  top: -$tech-card-box-overlap;
+  width: 100%;
+  z-index: -1;
+}
+
+.tech-card {
+  align-items: center;
+  background: $earls-green;
+  border-radius: $tech-card-border-radius;
+  display: flex;
+  flex-direction: column;
+  left: $tech-card-box-overlap;
+  min-height: $tech-card-box-min-height; 
+  padding: 10px 20px;
+  position: relative;
+  top: $tech-card-box-overlap;
+  width: $tech-card-box-width;
+}
+
+.tech-card:hover:before {
```

Comment:
use ::before

Reply:
the gif might not be that explicit but there's a transition between the 2 states. check the trello card which has the documentation and add it